### PR TITLE
fix: allow autocomplete of usernames

### PR DESCRIPTION
**Pull request by quad**
## Description:

tl;dr change the sign on “username” field’s `autocomplete` attribute to `username` instead of `identifier`?

- `autocomplete=identifier` doesn’t trigger password autocomplete in some environments (e.g. iOS Safari, 1Password)
- Moreover, `identifier` autocompletes in some environments with a person’s name (e.g. Scott Robinson)
- `identifier` is not in the [WhatWG autofill standard](https://html.spec.whatwg.org/multipage/forms.html#autofill)

_(I wasn't able to read the test guidelines or security best practices as they're locked behind a Confluence login.)_

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [x] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- #1119
- [OKTA-552271](https://oktainc.atlassian.net/browse/OKTA-552271)
- [OKTA-293139](https://oktainc.atlassian.net/browse/OKTA-293139)

### CODE REVIEW

**Comment by denysoblohin-okta** [comment]
Thanks for posting this PR.
Seems reasonable to change `autocomplete` attribute to the standard one.
Internal ref: OKTA-552271

**Comment by quad** [comment]
@denysoblohin-okta, you're very welcome! What is the next step to getting this PR merged?

**Comment by denysoblohin-okta** [comment]
I created a [copy PR](https://github.com/okta/okta-signin-widget/pull/2952) to pass internal CI tests

**Review by denysoblohin-okta — APPROVED** [review]

**Comment by quad** [comment]
> I created a [copy PR](https://github.com/okta/okta-signin-widget/pull/2952) to pass internal CI tests

Yay! Should we close this one in favour of yours?

**Comment by quad** [comment]
oh wow, JINX
